pontoon/api/schema.py

```diff
@@ -0,0 +1,118 @@
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django.debug import DjangoDebug
+
+from .util import get_fields
+
+from ..base.models import (
+    Project as ProjectModel,
+    Locale as LocaleModel,
+    ProjectLocale as ProjectLocaleModel
+)
+
+
+class Stats(graphene.AbstractType):
+    missing_strings = graphene.Int()
+    complete = graphene.Boolean()
+
+
+class ProjectLocale(DjangoObjectType, Stats):
+    class Meta:
+        model = ProjectLocaleModel
+        only_fields = (
+            'total_strings', 'approved_strings', 'translated_strings',
+            'fuzzy_strings', 'project', 'locale'
+        )
+
+
+class Project(DjangoObjectType, Stats):
+    class Meta:
+        model = ProjectModel
+        only_fields = (
+            'name', 'slug', 'info', 'deadline', 'priority', 'contact',
+            'total_strings', 'approved_strings', 'translated_strings',
+            'fuzzy_strings'
+        )
+
+    localizations = graphene.List(ProjectLocale)
+
+    @graphene.resolve_only_args
+    def resolve_localizations(obj):
+        return obj.project_locale.all()
+
+
+class Locale(DjangoObjectType, Stats):
+    class Meta:
+        model = LocaleModel
+        only_fields = (
+            'name', 'code', 'direction', 'script', 'population',
+            'total_strings', 'approved_strings', 'translated_strings',
+            'fuzzy_strings'
+        )
+
+    localizations = graphene.List(ProjectLocale)
+
+    @graphene.resolve_only_args
+    def resolve_localizations(obj):
+        return obj.project_locale.all()
+
+
+class Query(graphene.ObjectType):
+    debug = graphene.Field(DjangoDebug, name='__debug')
+
+    projects = graphene.List(Project)
+    project = graphene.Field(Project, slug=graphene.String())
```

Comment:
nit and curiosity: I wonder what's the convention -> I saw that a few random internet pages capitalize all fields in their GraphQL examples.

Reply:
Looking at the example on http://graphql.org/, the convention is to capitalize shape types and use all-lowercase for field names.

Reply:
That would be coherent with PEP8's naming conventions, so I like it. Let's not be weird. :)